src/interval_sets.jl

```diff
@@ -521,3 +521,55 @@ function find_intersections_helper!(result, x, y, lt)
 
     return unique!.(result)
 end
+
+function find_intersections(
+    x::AbstractVector{<:AbstractInterval{T1,Closed,Closed}},
+    y::AbstractVector{<:AbstractInterval{T2,Closed,Closed}},
+) where {T1,T2}
```

Comment:
Currently only executed by a single test. That makes me worried... I'll see what I can do about this

Reply:
there's a few in `test/sets.jl`, def not just 1 (since they caught various bugs when developing this)

Reply:
This method is called 6 times in the `test/sets.jl` file (I inserted a global, incremented it in this method, then checked before-vs-after running `test/sets.jl`.